### PR TITLE
Added Zombie Kidz Evolution

### PR DESCRIPTION
**Pull request by chinmay7016**
#269 
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by chinmay7016** [comment]
@edm00se if you like this changes pls merge this pr

File: readme.md [hunk 1]
```diff
@@ -2517,6 +2517,20 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 |-------: | -------: | ------:|
 |   2 - 5 |       13 |  90m   |
 
+### [zombie-kidz-evolution](https://boardgamegeek.com/boardgame/256952/zombie-kidz-evolution)
```
Comment (edm00se) [inline readme.md:2520]: Please fix the title.

File: readme.md [hunk 2]
```diff
@@ -2517,6 +2517,20 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+
+> Zombie Kidz Evolution is the first "legacy"-style game for kids, with basic gameplay similar to the 2013 title Zombie Kidz — but things will quickly evolve!
+
+In the game, the young players are at school when zombies start to attack, infiltrating the entryways, then barring the way to classrooms. Players move through rooms and entryways to eliminate zombies and keep the zombie reserve full. If a zombie must enter the school but none are available, then the school has been overrun and the players lose. To win the game, players need to lock each of the four entryways as apparently the custodian has abandoned the post and given up the school for lost.
```
Comment (edm00se) [inline readme.md:2524]: Please update the description block formatting here as well.

**Review by edm00se — CHANGES_REQUESTED** [review]
Description block formatting and the title needs to be fixed.

**Comment by chinmay7016** [comment]
@edm00se  sir pls review changes and merge it pls sir.

File: readme.md [hunk 3]
```diff
@@ -2517,6 +2517,16 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 |-------: | -------: | ------:|
 |   2 - 5 |       13 |  90m   |
 
+### [zombie kidz evolution](https://boardgamegeek.com/boardgame/256952/zombie-kidz-evolution)
```
Comment (edm00se) [inline readme.md:2520]: The title should be cased correctly.

```suggestion
### [Zombie Kidz Evolution](https://boardgamegeek.com/boardgame/256952/zombie-kidz-evolution)
```

**Review by edm00se — APPROVED** [review]

**Pull request merged**